### ISSUE TITLE
GET endpoints - S3 bucket does not exist

### ISSUE THREAD

**Issue by Animesh-1983** [ryanmurakami/trash-panda-buffet#12]
Hi Ryan,  Thanks for putting thru a good trail of courses for AWS developer cert. I am now in serverless but facing below issue when trying to execute GET endpoint after "npm run deploy : aws" -

message | "The specified bucket does not exist"
code | "NoSuchBucket"
region | null
time | "2020-10-03T05:37:26.123Z"
requestId | "50D25AADCFD6B57E"
extendedRequestId | "ldAGjyHlxNe9FkF1vD8POp9dFUELmtoQsHU7s1BXLLeJnLtPND5NGKQfd2zDuKKck2SOdnVmm24="
statusCode | 404
retryable | false
retryDelay | 85.59063296187126

I am not getting this error while deploying the application - though I am also not seeing a "success" msg after deployment  ends. Below is the log from command prompt (I am using a desktop with windows10) for my deployment:

> trash-panda-buffet@2.0.0 deploy:aws C:\Users\91779\Documents\trash-panda-buffet
> serverless deploy

Serverless: Packaging service...
Serverless: Excluding development dependencies...
Serverless: Uploading CloudFormation file to S3...
Serverless: Uploading artifacts...
Serverless: Uploading service trash-panda-buffet.zip file to S3 (15.59 MB)...
Serverless: Validating template...
Serverless: Updating Stack...
Serverless: Checking Stack update progress...
....................................................................
Serverless: Stack update finished...
Service Information
service: trash-panda-buffet
stage: dev
region: ap-south-1
stack: trash-panda-buffet-dev
resources: 62
api keys:
  None
endpoints:
  GET - https://qxpiw78893.execute-api.ap-south-1.amazonaws.com/dev/
  POST - https://qxpiw78893.execute-api.ap-south-1.amazonaws.com/dev/game
  PUT - https://qxpiw78893.execute-api.ap-south-1.amazonaws.com/dev/game/{id}
  GET - https://qxpiw78893.execute-api.ap-south-1.amazonaws.com/dev/scores
  GET - https://qxpiw78893.execute-api.ap-south-1.amazonaws.com/dev/foodItems
  GET - https://qxpiw78893.execute-api.ap-south-1.amazonaws.com/dev/trashPanda
functions:
  index: trash-panda-buffet-dev-index
  gameStart: trash-panda-buffet-dev-gameStart
  gameSave: trash-panda-buffet-dev-gameSave
  gameScores: trash-panda-buffet-dev-gameScores
  gameVerify: trash-panda-buffet-dev-gameVerify
  populateData: trash-panda-buffet-dev-populateData
  getFoodItems: trash-panda-buffet-dev-getFoodItems
  getRandomTrashPanda: trash-panda-buffet-dev-getRandomTrashPanda
  saveAction: trash-panda-buffet-dev-saveAction
  processAction: trash-panda-buffet-dev-processAction
layers:
  None
DistributionIdKey: cloudFrontDistributionOutput
CloudfrontInvalidate: Invalidation started

**Comment by ryanmurakami** [comment]
Hi Animesh,

I'm sorry, I don't understand the issue. The output you sent from `npm deploy` seems to be correct. When do you experience an error?

**Comment by joeratt** [comment]
I ran into this same issue, so figured I'd share how I solved it.

Tl;dr: My issue resulted during the `npm run build:client` stage, which occurs after the aws step. `@babel/preset-env` had an issue, and I had to update it to `"7.12.1"` 
* Related Issue: https://github.com/babel/babel/issues/11615


**Error Block:**
```JavaScript
Serverless: Stack update finished...
Service Information
service: trash-panda-buffet
stage: dev
region: us-east-2
stack: trash-panda-buffet-dev
resources: 62
api keys:
  None
endpoints:
  GET - https://h13lvp714l.execute-api.us-east-2.amazonaws.com/dev/
  POST - https://h13lvp714l.execute-api.us-east-2.amazonaws.com/dev/game
  PUT - https://h13lvp714l.execute-api.us-east-2.amazonaws.com/dev/game/{id}
  GET - https://h13lvp714l.execute-api.us-east-2.amazonaws.com/dev/scores
  GET - https://h13lvp714l.execute-api.us-east-2.amazonaws.com/dev/foodItems
  GET - https://h13lvp714l.execute-api.us-east-2.amazonaws.com/dev/trashPanda
functions:
  index: trash-panda-buffet-dev-index
  gameStart: trash-panda-buffet-dev-gameStart
  gameSave: trash-panda-buffet-dev-gameSave
  gameScores: trash-panda-buffet-dev-gameScores
  gameVerify: trash-panda-buffet-dev-gameVerify
  populateData: trash-panda-buffet-dev-populateData
  getFoodItems: trash-panda-buffet-dev-getFoodItems
  getRandomTrashPanda: trash-panda-buffet-dev-getRandomTrashPanda
  saveAction: trash-panda-buffet-dev-saveAction
  processAction: trash-panda-buffet-dev-processAction
layers:
  None
DistributionIdKey: cloudFrontDistributionOutput
CloudfrontInvalidate: Invalidation started
Serverless: Removing old service artifacts from S3...

****************************************************************************************************************************
Serverless: Announcing an enhanced experience for running Express.js apps: https://github.com/serverless-components/express.
****************************************************************************************************************************


> trash-panda-buffet@2.0.0 deploy:client
> npm run build:client && serverless client deploy --no-confirm


> trash-panda-buffet@2.0.0 build:client
> webpack --config ./src/client/webpack.prod.config.js

Hash: 68c3d904062e7124b42e
Version: webpack 4.41.5
Time: 503ms
Built at: 11/13/2020 3:11:25 PM
 1 asset
Entrypoint main = main.js
[0] ./src/client/index.js 5.83 KiB {0} [built] [failed] [1 error]

ERROR in ./src/client/index.js
Module build failed (from ./node_modules/babel-loader/lib/index.js):
Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: No "exports" main defined in /Users/<user>/git/aws/aws-developer-serverless-architecture-monitoring/02/demos/before/node_modules/@babel/helper-compilation-targets/package.json
    at new NodeError (node:internal/errors:277:15)
    at throwExportsNotFound (node:internal/modules/esm/resolve:321:9)
    at packageExportsResolve (node:internal/modules/esm/resolve:546:3)
    at resolveExports (node:internal/modules/cjs/loader:467:36)
    at Function.Module._findPath (node:internal/modules/cjs/loader:507:31)
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:905:27)
    at Function.Module._load (node:internal/modules/cjs/loader:763:27)
    at Module.require (node:internal/modules/cjs/loader:991:19)
    at require (/Users/<user>/git/aws/aws-developer-serverless-architecture-monitoring/02/demos/before/node_modules/v8-compile-cache/v8-compile-cache.js:161:20)
    at Object.<anonymous> (/Users/<user>/git/aws/aws-developer-serverless-architecture-monitoring/02/demos/before/node_modules/@babel/preset-env/lib/debug.js:8:33)
    at Module._compile (/Users/<user>/git/aws/aws-developer-serverless-architecture-monitoring/02/demos/before/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1131:10)
    at Module.load (node:internal/modules/cjs/loader:967:32)
    at Function.Module._load (node:internal/modules/cjs/loader:807:14)
    at Module.require (node:internal/modules/cjs/loader:991:19)
    at require (/Users/<user>/git/aws/aws-developer-serverless-architecture-monitoring/02/demos/before/node_modules/v8-compile-cache/v8-compile-cache.js:161:20)
    at Object.<anonymous> (/Users/<user>/git/aws/aws-developer-serverless-architecture-monitoring/02/demos/before/node_modules/@babel/preset-env/lib/index.js:11:14)
    at Module._compile (/Users/<user>/git/aws/aws-developer-serverless-architecture-monitoring/02/demos/before/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1131:10)
    at Module.load (node:internal/modules/cjs/loader:967:32)
    at Function.Module._load (node:internal/modules/cjs/loader:807:14)
    at Module.require (node:internal/modules/cjs/loader:991:19)
    at require (/Users/<user>/git/aws/aws-developer-serverless-architecture-monitoring/02/demos/before/node_modules/v8-compile-cache/v8-compile-cache.js:161:20)
    at requireModule (/Users/<user>/git/aws/aws-developer-serverless-architecture-monitoring/02/demos/before/node_modules/@babel/core/lib/config/files/plugins.js:165:12)
    at loadPreset (/Users/<user>/git/aws/aws-developer-serverless-architecture-monitoring/02/demos/before/node_modules/@babel/core/lib/config/files/plugins.js:83:17)
    at createDescriptor (/Users/<user>/git/aws/aws-developer-serverless-architecture-monitoring/02/demos/before/node_modules/@babel/core/lib/config/config-descriptors.js:154:9)
    at /Users/<user>/git/aws/aws-developer-serverless-architecture-monitoring/02/demos/before/node_modules/@babel/core/lib/config/config-descriptors.js:109:50
    at Array.map (<anonymous>)
    at createDescriptors (/Users/<user>/git/aws/aws-developer-serverless-architecture-monitoring/02/demos/before/node_modules/@babel/core/lib/config/config-descriptors.js:109:29)
    at createPresetDescriptors (/Users/<user>/git/aws/aws-developer-serverless-architecture-monitoring/02/demos/before/node_modules/@babel/core/lib/config/config-descriptors.js:101:10)
    at /Users/<user>/git/aws/aws-developer-serverless-architecture-monitoring/02/demos/before/node_modules/@babel/core/lib/config/config-descriptors.js:58:104
    at cachedFunction (/Users/<user>/git/aws/aws-developer-serverless-architecture-monitoring/02/demos/before/node_modules/@babel/core/lib/config/caching.js:62:27)
    at cachedFunction.next (<anonymous>)
    at evaluateSync (/Users/<user>/git/aws/aws-developer-serverless-architecture-monitoring/02/demos/before/node_modules/gensync/index.js:244:28)
    at sync (/Users/<user>/git/aws/aws-developer-serverless-architecture-monitoring/02/demos/before/node_modules/gensync/index.js:84:14)
    at presets (/Users/<user>/git/aws/aws-developer-serverless-architecture-monitoring/02/demos/before/node_modules/@babel/core/lib/config/config-descriptors.js:29:84)
    at mergeChainOpts (/Users/<user>/git/aws/aws-developer-serverless-architecture-monitoring/02/demos/before/node_modules/@babel/core/lib/config/config-chain.js:320:26)
    at /Users/<user>/git/aws/aws-developer-serverless-architecture-monitoring/02/demos/before/node_modules/@babel/core/lib/config/config-chain.js:283:7
    at Generator.next (<anonymous>)
    at buildRootChain (/Users/<user>/git/aws/aws-developer-serverless-architecture-monitoring/02/demos/before/node_modules/@babel/core/lib/config/config-chain.js:68:36)
    at buildRootChain.next (<anonymous>)
    at loadPrivatePartialConfig (/Users/<user>/git/aws/aws-developer-serverless-architecture-monitoring/02/demos/before/node_modules/@babel/core/lib/config/partial.js:95:62)
    at loadPrivatePartialConfig.next (<anonymous>)
    at Function.<anonymous> (/Users/<user>/git/aws/aws-developer-serverless-architecture-monitoring/02/demos/before/node_modules/@babel/core/lib/config/partial.js:120:25)
    at Generator.next (<anonymous>)
    at evaluateSync (/Users/<user>/git/aws/aws-developer-serverless-architecture-monitoring/02/demos/before/node_modules/gensync/index.js:244:28)
    at Function.sync (/Users/<user>/git/aws/aws-developer-serverless-architecture-monitoring/02/demos/before/node_modules/gensync/index.js:84:14)
    at Object.<anonymous> (/Users/<user>/git/aws/aws-developer-serverless-architecture-monitoring/02/demos/before/node_modules/@babel/core/lib/config/index.js:41:61)
npm ERR! code 2
```
